docs/plugins/from-4-to-5.rst

```diff
@@ -0,0 +1,118 @@
+Plugin upgrade from 4 to 5 step by step
+=======================================
+
+Here is a list of steps that most of the Plugins may need to take to upgrade from Mautic 4 to Mautic 5. You should be able to get through each step, make a commit, move to the next one and once you are at finished you have upgraded your Plugin.
+
+1) Continuous Integration
+-------------------------
+
+If you don't have CI configured, this is the time to do it. This is an optional step but it makes sense to do it at the beginning rather than later. Here's how to get it done: https://devdocs.mautic.org/en/5.x/plugins/continuous-integration.html
```

Comment:
Let's use the built-in referencing tools:

```suggestion
If you don't have CI configured, this is the time to do it. This is an optional step but it makes sense to do it at the beginning rather than later. Here's how to get it done: :doc:`/plugins/continuous-integration`.
```